scripts/start-gitlab.sh

```diff
@@ -11,7 +11,7 @@ then
         exit 1
     fi
 else
-    img=gitlab/gitlab-ce
+    img=gitlab/gitlab-ce:13.0.6-ce.0
```

Comment:
Work-around for https://github.com/terraform-providers/terraform-provider-gitlab/issues/338#issuecomment-647694847

Reply:
The deploy key issue was resolved by Gitlab 13.1.1 so I reverted this change but this now JIRA service tests fail:

```sh
# MAKE_TARGET=testacc scripts/start-gitlab.sh
Starting gitlab container...
dfd4ea69b610401cf74ff8a1f5d2803d0ee460ef2bc2a4679925db15b5746869
Waiting for gitlab to be ready ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... .
Creating access token



--------------------------------------------------------------------------------
 GitLab:       13.1.1 (bdb9883705a) FOSS
 GitLab Shell: 13.3.0
 PostgreSQL:   11.7
--------------------------------------------------------------------------------
Loading production environment (Rails 6.0.3.1)
Switch to inspect mode.
terraform_token = PersonalAccessToken.create(user_id: 1, scopes: [:api, :read_user], name: :terraform);terraform_token.set_token('ACCTEST');terraform_token.save!;
true

# make testacc TEST=./gitlab TESTARGS='-run TestAccGitlabServiceJira_basic -count 1'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test -v ./gitlab -run TestAccGitlabServiceJira_basic -count 1 -timeout 40m
=== RUN   TestAccGitlabServiceJira_basic
    TestAccGitlabServiceJira_basic: testing.go:684: Step 1 error: Check failed: Check 2/8 error: gitlab_service_jira.jira: Attribute 'url' expected "https://testurl.com", got "https://test.com"
--- FAIL: TestAccGitlabServiceJira_basic (10.67s)
FAIL
FAIL    github.com/terraform-providers/terraform-provider-gitlab/gitlab 10.684s
FAIL
make: *** [GNUmakefile:23: testacc] Error 1
```

https://github.com/claranet/terraform-provider-gitlab/runs/810546410

Reply:
The pinning was reinstated to work this around for now.

Reply:
Removed the pinning as the issue mysteriously no longer occurs: https://github.com/terraform-providers/terraform-provider-gitlab/issues/353